lib/config.py

```diff
@@ -10,6 +10,8 @@
 )
 sentinel_config_file = os.environ.get('SENTINEL_CONFIG', default_sentinel_config)
 sentinel_cfg = DashConfig.tokenize(sentinel_config_file)
+sentinel_version = 2
```

Comment:
I think this should be fine for now, but we might wanna discuss a versioning scheme for Sentinel and how we wanna handle it.